resources/views/platform/components/game/achievements-list-meta.blade.php

```diff
@@ -0,0 +1,35 @@
+@props([
+    'isOfficial' => true,
+    'numAchievements' => 0,
+    'numMissableAchievements' => 2,
+    'totalPossible' => 0, // total possible points
+    'totalPossibleTrueRatio' => 0, // total possible RetroPoints
+])
+
+<div x-data="toggleAchievementRowsComponent()" class="flex flex-col gap-y-1">
+    <p>
+        There {{ $numAchievements === 1 ? 'is' : 'are'}}
+        <span class="font-bold">
+            {{ localized_number($numAchievements) }}
+            @if (!$isOfficial)
+                Unofficial
+            @endif
+        </span>
+        {{ mb_strtolower(__res('achievement', $numAchievements)) }}
+        worth
+        <span class="font-bold">
+            {{ localized_number($totalPossible) }}
+        </span>
+        <x-points-weighted-container>({{ localized_number($totalPossibleTrueRatio )}})</x-points-weighted-container>
+        {{ mb_strtolower(__res('point', $totalPossible)) }}.
+    </p>
+
+    @if ($numMissableAchievements > 0)
+        <button class="flex items-center gap-x-1 hover:underline hover:text-link-hover text-link" @click="toggleNonMissableRows">
+            <div class="w-5 h-5 p-0.5 rounded-full bg-embed border border-dashed border-stone-500 text-white">
+                <x-icon.missable />
+            </div>
+            This set has {{ $numMissableAchievements }} missable {{ mb_strtolower(__res('achievement', $numMissableAchievements)) }}.
+        </button>
```

Comment:
I don't really link this button style. It looks like a link, which says to me that it'll open a new page.

Reply:
When it takes a standard `.btn` classname, it becomes fairly loud on the page. Maybe we need a small meta surface above the achievements list which contains filtering / sorting options similar to other pages (ie: completion progress, most requested sets, recent game awards, etc).

Reply:
In latest, this is now using a `.btn` class.

![Screenshot 2023-12-05 at 6 00 12 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/b220d5d7-12d1-4a18-8b0f-849bd56584ff)

On clicking the button, the filter is applied and the text changes informing the user they can click again to reset the view.

I do think we should rework all the filtering and sorting using Livewire, but it's a fairly large undertaking likely beyond the scope of this PR.

Reply:
I agree that a better filtering system should be a separate PR, but I'm still not comfortable putting behavior on the summary detail. What about adding a second check mark?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0774dee8-732e-4e9d-a4fc-ce393eaa6f78)

Reply:
Or maybe separate the button from the summary?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/914fa617-dab5-4129-aec8-e73e1099235b)

But at that point, the checkbox feels more consistent.

Reply:
I think I like the checkbox most, as it keeps the sorting/filtering function in the same centralized location.